core/controllers/editor.py

```diff
@@ -781,19 +768,10 @@ class TopUnresolvedAnswersHandler(EditorHandler):
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
 
     @acl_decorators.can_edit_exploration
-    def get(self, exploration_id):
+    def get(self, unused_exploration_id):
         """Handles GET requests for unresolved answers."""
-        state_name = self.request.get('state_name')
-        if not state_name:
-            raise self.PageNotFoundException
-
-        unresolved_answers_with_frequency = (
-            stats_services.get_top_state_unresolved_answers(
-                exploration_id, state_name))
-
-        self.render_json({
-            'unresolved_answers': unresolved_answers_with_frequency
-        })
+        # TODO(#11475): Return visualizations info based on Apache Beam job.
+        self.render_json({'unresolved_answers': []})
```

Comment:
So currently this API will return no value?  is there any other way to retrieve this data? [Also, I don't see a clear action item for fixing this handler in the TODO issue [am I missing anything?]]

Reply:
The entirety of stats needs to be rehauled, so most if not all of these models will become deprecated. I don't have time to outline how, however, so I'm fine with leaving "stubbed" APIs with that in mind.